awesomplete.js

```diff
@@ -95,7 +95,7 @@ var _ = function (input, o) {
 				li = li.parentNode;
 			}
 
-			if (li) {
+			if (li && evt.which === 1) {  // Only select on left click
```

Comment:
My bad. 

I guess we should probably go with the standard and less convoluted `evt.button === 0` which [seem to be well supported](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button).